uber/site_sections/preregistration.py

```diff
@@ -113,23 +113,26 @@ def form(self, session, message='', edit_id=None, **params):
         """
         params['id'] = 'None'   # security!
 
+        # Both the Attendee class and Group class have identically named
+        # address fields. In order to distinguish the two sets of address
+        # fields in the params, the Group fields are prefixed with "group_"
+        # when the form is submitted. To prevent instantiating the Group object
+        # with the Attendee's address fields, we must clone the params and
+        # rename all the "group_" fields.
+        group_params = dict(params)
```

Comment:
But if we change those pages, we have to also change their functions, which is a different kind of brittle... Wait, that means this block being here isn't an issue, because we WILL always use the group_ prefix in the pre-registration form. I think. I'm pretty sure.

To answer your question though, it's the group management page (group_members.html) and the admin form for groups.